src/main/java/jp/co/yahoo/yosegi/block/PushdownSupportedBlockWriter.java

```diff
@@ -244,47 +245,42 @@ public int size() {
 
     columnTree.create( metaBuffer , dataBuffer );
 
-    byte[] metaBinary = compressor.compress( metaBuffer.getBytes() , 0 , metaBuffer.getLength() );
-
-    byte[] result;
-    if ( dataSize == -1 ) {
-      result = new byte[
-          headerBytes.length
-          + dataBuffer.getLength()
-          + metaBinary.length
```

Comment:
I think the following data size seems not to write.
dataBuffer.getLength() + metaBinary.length

Reply:
Lines 269 and 271 write to OutputStream.
If this process is not written, unit test fails because data can not be read.

Reply:
Thank you.